wrappers/services.go

```diff
@@ -42,46 +44,150 @@ import (
 	"github.com/snapcore/snapd/wrappers/internal"
 )
 
+// ServiceScope represents the scope of services that can be present in a snap, which
```

Comment:
s/represents/indicates/ perhaps? same in the comments below?

Reply:
nice suggestion, changed